src/_scss/pages/search/results/visualizations/geo/map/_toggle.scss

```diff
@@ -88,10 +88,28 @@
                         top: rem(62) !important;
                         left: rem(-30) !important;
                     }
-                    @media ((min-width: $tablet-screen) and (max-width: $large-screen)) {
+                    @media ((min-width: $tablet-screen) and (max-width: $medium-screen)) {
+                        top: rem(40) !important;
+                        left: rem(-30) !important;
+
+                        .smart-bottom-right {
+                            left: 24.3rem;
+                            
+                        }
+                    }
+                    @media ((min-width: $medium-screen) and (max-width: $large-screen)) {
```

Comment:
these two media queries can be combined into one, since they use the same rules; between tablet and large screen sizes